GiveNGo/GiveNGo
# Router and Navegation

## Description:
Implemented nested router and navigation between screens with Router Navigation. Implemented Explore feature UI.
 
## Changes I Made:
- AboutUs.tsx: created about us screen templet need to add content  
- App.tsx: implemented stack navigation and deleted inline style 
- Index.ts: register app
- Package.json: installed navigations library and maps library 
- Create Routes Folder
- Footer.tsx: created foot navagation bar
- DashboardRouter.tsx, ExploreRouter.tsx, RequestRouter.tsx, LoginRouter.tsx: created top navigation bar
- ChatPage.tsx, ExplorePage.tsx, RequestPage.tsx, LoginPage.tsx, DashboardPage.tsx, CodeOfConductPage.tsx, ProfilePage.tsx and SignUpPage.tsx: deleted old Footer and NavBar references
- ExplorePage.tsx and RequestPin.tsx: implanted UI for map view need to add server connection for real data

## How to Test:
npm start or yarn start, press any button on navigations bars to see screen change
